.github/scripts/trymerge.py

```diff
@@ -2292,6 +2299,17 @@ def handle_exception(e: Exception, title: str = "Merge failed") -> None:
         )
         return
 
+    if args.check_mergeability:
+        if pr.is_ghstack_pr():
+            get_ghstack_prs(repo, pr)  # raises error if out of sync
+        pr.merge_changes(
+            repo,
+            branch="main",
+            skip_mandatory_checks=True,
```

Comment:
Nit: IMO, having 2 params called `skip_mandatory_checks` and `skip_all_rule_checks` is a bit weird as the latter's name implies that it subsumes the former.  For example, in this call, setting `skip_mandatory_checks` to `True` or `False` doesn't matter.  I don't want it to give the impression that `skip_all_rule_checks` is an option to bring in a bigger force merge hammer.  It's more like a different kind of `--dry-run` like what you call out in the description.